web/client/epics/featuregrid.js

```diff
@@ -470,6 +483,17 @@ module.exports = {
         .switchMap( () => {
             return Rx.Observable.of(setControlProperty("drawer", "enabled", false), toggleTool("featureCloseConfirm", false));
         }),
+    /**
+     * Close sync filter wms control on faturegrid close,
+     * not on advanced search open
+     */
+     toggleSyncControl: (action$, store) =>
+         action$.ofType(OPEN_ADVANCED_SEARCH, CLOSE_FEATURE_GRID)
+            .filter(() => isSyncWmsActive(store.getState()))
+            .scan((acc, cur) => {
+                return cur.type === CLOSE_FEATURE_GRID && acc.type !== OPEN_ADVANCED_SEARCH ? toggleControl("syncwmsfilter") : cur;
```

Comment:
This intercepts all the : 
 - CLOSE_FEATURE_GRID --> CLOSE_FEATURE_GRID
Instead you want to stop sync on feature grid close. You should see the action ASK_CLOSE_FEATURE_GRID_CONFIRM is triggered on close. There is an epic that intercepts this and follows a close grid, if no confirm is required. If the user see the confirm and deny it emits FEATUREGRID:FEATURE_GRID_CLOSE_CONFIRMED (strange name, maybe let's check and change the name of this action later. 

You can intercept this sequence of actions: 
ASK_CLOSE_FEATURE_GRID_CONFIRM --> FEATUREGRID:CLOSE_GRID -->  take until FEATUREGRID:FEATURE_GRID_CLOSE_CONFIRMED
something like this should work: 
```js
      .filter( () => isSyncWmsActive(store.getState()))
      action$.ofType(ASK_CLOSE_FEATURE_GRID_CONFIRM)
      .switchMap(() => 
         action$.ofype(FEATUREGRID:CLOSE_GRID)
             .takeUntil(action$.ofype(FEATUREGRID:FEATURE_GRID_CLOSE_CONFIRMED))
             .take(1)
             .map(() => stopSync() )
```

You could make a function to use in less. Should be something like this:
```js
const featureGridIsClosed = ( action$ ) => 
 action$.ofType(ASK_CLOSE_FEATURE_GRID_CONFIRM)
      .switchMap(() => 
         action$.ofype(FEATUREGRID:CLOSE_GRID)
             .takeUntil(action$.ofype(FEATUREGRID:FEATURE_GRID_CLOSE_CONFIRMED))
             .take(1)
```

And use it with let
```js
   // .. intercept feature grid first opening
            .switchMap ( 
                action$.let(fetaureGridIsClosed).map(() => stopSync() )
```